third_party/move/testing-infra/transactional-test-runner/src/macros.masm.tera

```diff
@@ -0,0 +1,18 @@
+{#
+Copyright © Aptos Foundation
+SPDX-License-Identifier: Apache-2.0
+
+Tera templates for supporting Move ASM transactional tests.
+
+To use tera templates n
+#}
+
+
+{% macro expect(ty, val) -%}
```

Comment:
Should this be called `expect_number` or `expect_numeric` or similar, showcasing it is limited to numbers?

Reply:
Done